### ISSUE TITLE
Change name of Working Groups to Rings

### ISSUE THREAD

**Issue by bmann** [ethereum-magicians/scrolls#9]
Category is called Working Groups https://ethereum-magicians.org/c/working-groups -- we should change the name to Rings.

Also, the "about this category" could use editing and point to the wiki and such https://ethereum-magicians.org/t/about-the-working-groups-category/305

**Comment by ltfschoen** [comment]
Action required from forum user with sufficient privileges

**Comment by jpitts** [comment]
I actually was thinking of not doing this, as it sacrifices clarity for consistency. Having Working Groups there helps communicate to new people what a Ring is, without making it harder for those already participating in Magicians to find their Rings. 

Example:  "Rings: Data Ring" seems a lot more mysterious than "Working Groups: Data RIng".

**Comment by bmann** [comment]
I file issues to indicate that work needs to be done or a decision needs to be made. If it’s not assigned to anyone, no work gets done.

**Comment by jpitts** [comment]
LOL, "I actually was thinking of not doing this"... I should have been more clear! I don't think that we should do this.

But I'll update the Forum if y'all insist on calling "Working Groups" -> "Rings" :)

**Comment by bmann** [comment]
Nope, just leave it open. That was my point. We don’t have consensus. 

I wasn’t necessarily filing this on my own behalf. I don’t remember the context.
